test/test_pyvpgl.py

```diff
@@ -1,12 +1,111 @@
 import unittest
+import pickle
 
-try:
-  import numpy as np
-except:
```

Comment:
nitpick - ```except ImportError``` would be more explicit

Reply:
Nitpick understood (though this numpy check was removed...).  Will remember for future tests.